integration_test/ops_agent_test/main_test.go

```diff
@@ -4087,10 +4087,33 @@ func unmarshalResource(in string) (*resourcedetector.GCEResource, error) {
 	return &resource, err
 }
 
-// installGolang downloads and sets up go on the given VM.  The caller is still
+// uninstallGolang removes the go installation on the VM.
+func uninstallGolang(ctx context.Context, logger *log.Logger, vm *gce.VM) error {
+	var cmd string
+	if gce.IsWindows(vm.ImageSpec) {
+		// There is currently no need for this support.
+		return nil
+	} else {
+		cmd = `
+		sudo rm -rf /usr/local/go || true
```

Comment:
Since `rm` is run under `sudo`, when would it have a non-zero exit code?

Reply:
That's what the `-f` flag is for, right? It should not fail for nonexistent files.

Reply:
Removed